### DISCUSSION TITLE
project cells onto umap of ucsc cell browser datasets without the umap model

### DISCUSSION THREAD

**Discussion by yesonse** [satijalab/seurat#6880] [Q&A]
Any way to project query cells into the umap of ucsc cell browser dataset without informations of umap model?

I am able to create seurat projects from dataset of ucsc cell browser by following their suggestion.
I did not runUMAP on the project since it will make a new one different from the old umap.

I am able to runPCA and do Cell type classification by using the dataset downloaded with with TransferData .
I want to project the query cell to the umap in the reference too, but it failed with ProjectUMAP since no model kept.

Thanks

Robin

**Comment by Dragonmasterx87** [comment]
Hi @yesonse 

You can certainly run `RunUMAP` again. Just change the `reduction.name = "umap",  reduction.key = "UMAP_",` params. `reduction.name` is the name of the reduction slot to store your dimensional reduction under.

Cheers,

🐲
